libs/ngx-charts-on-fhir/src/lib/data-layer/data-layer-merge.service.ts

```diff
@@ -40,6 +40,12 @@ export class DataLayerMergeService {
         }
       }
     }
+    mergedLayer.datasets.sort((a: Dataset<TimelineChartType, TimelineDataPoint[]>, b: Dataset<TimelineChartType, TimelineDataPoint[]>) => {
```

Comment:
you don't need to specify the type parameters when they are the same as the defaults:
```suggestion
    mergedLayer.datasets.sort((a: Dataset, b: Dataset) => {
```

Reply:
Okay, I will update that.